### DISCUSSION TITLE
Model previews

### DISCUSSION THREAD

**Discussion by liksar34** [lllyasviel/Fooocus#2696] [Ideas]
Please add some kind of extension Civit.ai Helper🙏
Very not convenient to work with such a list of models, especially when they are more than 20 pieces, and can not remember what model and for what, on the preview at least more less would be clear. Or when you put the cursor on the model, the preview appears, as it is realized with styles.

**Comment by realm174** [comment]
Personally, I take notes when i experiment with different models, loras, etc.. it is indeed hard to keep track of everything, so having notes helps a lot, especially when you try to remember trigger words, or specific configurations.

**Comment by toutjavascript** [comment]
Hello
I have created this github repo to preview models and styles with prompts you can choose :
https://github.com/toutjavascript/FoooXus-Fooocus-Extender

This looks like 
![Capture d’écran 2024-04-07 à 17 28 00](https://github.com/lllyasviel/Fooocus/assets/30899600/856c7da6-07e3-407a-af53-c8c843e21520)
